toolkit/GeneratedScript/commands.sql

```diff
@@ -0,0 +1,24 @@

```

Comment:
How did you generate this?

Reply:
Also, it seems like a security issue for customers to execute this. We need to comment/document explaining how we guarantee this is not a security concern.

Reply:
Readme explains that project DllConvertToStream is used to generate this file.

Reply:
Customers can generate on their own instead of directly using this one.